whois-api/src/main/java/net/ripe/db/whois/api/httpserver/DoSUpdateFilter.java

```diff
@@ -0,0 +1,25 @@
+package net.ripe.db.whois.api.httpserver;
+
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+
+public class DoSUpdateFilter extends WhoisDoSFilter {
+
+    @Override
+    public void doFilter(final HttpServletRequest request, final HttpServletResponse response, final FilterChain chain) throws IOException, ServletException {
+        if (request != null) {
+            final String method = request.getMethod();
+            if (!"GET".equalsIgnoreCase(method)) {
```

Comment:
Invert the check, i.e. return early if the method IS a GET, it's easier to read and consistent with the rest of the code.

Reply:
Use `jakarta.ws.rs.HttpMethod.GET`

Reply:
Done